src/app/admin/admin.component.ts

```diff
@@ -45,9 +45,15 @@ export class AdminComponent implements OnInit {
 
   showSeats = false;
 
+  userData;
+
   constructor(public http: HttpClient, private apiService: ApiService) {}
 
-  ngOnInit(): void {}
+  ngOnInit() {
+    console.log(history.state);
```

Comment:
remove console logs everywhere in application

Reply:
removed logs everywhere in the app